unstructured_inference/inference/ordering.py

```diff
@@ -27,7 +27,7 @@ def order_layout(
     if len(layout) == 0:
         return []
 
-    layout.sort(key=lambda element: element.y1)
+    layout.sort(key=lambda element: (element.y1, element.x1))
```

Comment:
Should this logic be moved to the unstructured repo anyways?

Reply:
Since we already have this logic in the unstructured repo: [unstructured/partition/pdf.py#L291](https://github.com/Unstructured-IO/unstructured/blob/main/unstructured/partition/pdf.py#L291)
```python
        sorted_page_elements = sorted(
            page_elements,
            key=lambda el: (
                el.coordinates[0][1] if el.coordinates else float("inf"),
                el.coordinates[0][0] if el.coordinates else float("inf"),
            ),
        )
```
it would benefit us to have a dedicated method to support sorting all page elements to maintain consistency across strategies.